packages/ui/test/pages/Profile.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import { Profile } from '../../src/pages/Profile/Profile'
+
+describe('UI: Profile', () => {
+  it('Shows balance', () => {
+    render(<Profile />)
```

Comment:
I recommend react testing library, but that can come later.

Reply:
Sure - I was using the same setup as a dapp boileplate. I'll consult with you later what you're using 👍